src/components/ToggleMap.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { ThemeProvider } from 'styled-components/macro';
+import { ButtonStyle } from './Button';
+
+const theme = {
+  fg: '#2C3742',
+  bg: 'white'
+};
+
+const invertTheme = ({ fg, bg }) => ({
+  fg: bg,
+  bg: fg
+});
+
+const toggleMap = props => (
```

Comment:
For readability, you should consider `destructuring` the arguments in this function component. Try the following:

```js
const toggleMap = ({toggle}={}) => (
  ...
        <ButtonStyle onClick={toggle} n>
```